.github/workflows/update-anchore-dependencies.yml

```diff
@@ -0,0 +1,49 @@
+name: PR to update Anchore dependencies
```

Comment:
I think this workflow just goes away, and there's a different workflow to run `grype/validations.yml` against a syft change in a syft PR.

Reply:
The goal is to open these pre-release PRs as integration branches in all repos during a release day, starting with pointing at integration branches for all dependencies with the PR in a draft state, then once releases start (after all pre-release PRs are green) we bump all dependencies to released versions and promote the PR from draft to mergable.